runtime/doc/lsp.txt

```diff
@@ -1047,10 +1050,11 @@ enable({name}, {enable})                                    *vim.lsp.enable()*
 
     Parameters: ~
       • {name}    (`string|string[]`) Name(s) of client(s) to enable.
-      • {enable}  (`boolean?`) `true|nil` to enable, `false` to disable
-                  (actively stops and detaches clients as needed, and force
-                  stops them if necessary after `client.exit_timeout`
-                  milliseconds)
+      • {enable}  (`boolean?`) If `true|nil`, enables auto-activation of the
+                  given LSP config on current and future buffers. If `false`,
+                  disables auto-activation and stops related LSP clients and
+                  servers (and may force-stop servers after `exit_timeout`
```

Comment:
```suggestion
                  servers (and defaults to force-stoping the server after `exit_timeout`
```

Reply:
Users can discover that `exit_timeout` decides the default. Otherwise don't need to over-explain here.